### PR TITLE
3.0 release fixes

### PR DESCRIPTION
**Pull request by piotrgrundas**
I want to merge this change because it adjustes storefront to 3.0 API
[SDK PR](https://github.com/mirumee/saleor-sdk/pull/105)

### Pull Request Checklist

1. [x] All visible strings are translated with proper context.
1. [x] All data-formatting is locale-aware (dates, numbers, and so on).
1. [x] The changes are tested.
1. [x] The code is documented (docstrings, project documentation).
1. [ ] Changes are mentioned in the changelog.

### Test Environment Config

API_URI=https://qa.staging.saleor.cloud/graphql/

### CODE REVIEW

File: package.json
Comment (dominik-zeglen) [inline package.json:11]: I think it should be pinned to a commit, so we we can be sure dependencies are stable. Or do you plan to release sdk before merging this pr?

Comment (piotrgrundas) [reply package.json:11]: @dominik-zeglen Yes, waiting for release.

**Review by bmigirl — APPROVED** [review]

**Review by jwm0 — APPROVED** [review]

**Pull request merged**
